tests/data/comments_non_breaking_space.py

```diff
@@ -0,0 +1,271 @@
+from .config import (
+    Any,
```

Comment:
Could you remove some extraneous code here? I think most of the imports on this line can go for example, and most of the cases in `class C` below aren't really relevant either. I'd want this test file to have only comments with nbsps and elements that are directly related (e.g., to test behavior around moving comments).